data/product.d/rhel.conf

```diff
@@ -61,6 +61,12 @@ default_help_pages =
     rhel_help_placeholder.xml
     rhel_help_placeholder.xml
 
+
+# The help system is currently disabled
```

Comment:
Maybe add a FIXME here?

Reply:
Added (and also fixed the the condition to use properly `help_dir` in rhel.conf :P).